deploy/kubernetes/filebeat-kubernetes.yaml

```diff
@@ -112,9 +112,16 @@ metadata:
 data:
   filebeat.yml: |-
     filebeat.inputs:
-    - type: container
+    - type: filestream
```

Comment:
@MichaelKatsoulis
shouldn't here be defined `id` in input? or in such case it will be automatically generated?

[doc](https://www.elastic.co/guide/en/beats/filebeat/current/filebeat-input-filestream.html):
```
Each filestream input must have a unique ID. Omitting or changing the filestream ID may cause data duplication. Without a unique ID, filestream is unable to correctly track the state of files.
```

so for all files that are matching `/var/log/containers/*.log` we have 1 filestream with unique id, correct? do you know what does it imply in comparison to the `autodiscover` where it is suggested to set filestream per container?

Reply:
@tetianakravchenko Yes an id will automatically get generated. When the `filebeat.input` is used instead of auto discovery then there will be one stream of `filestream` input looking at all files in the path. When autodiscovery is used there will be one stream for each discovered container looking at one log file only.

For the metadata in first scenario, the processor is used which  requires the matches log path so it can extract the container id from the log file name, and add the metadata of that container.
In the autodiscovery case the metadata are enriched by the kubernetes provider.

So yes, we have one filestream with one id for all the log collection. Both options work just fine. But with the first approach we cannot enable hints.